.devcontainer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM --platform=linux/amd64 mcr.microsoft.com/devcontainers/base:jammy
+
+SHELL ["/bin/bash", "--login", "-i", "-c"]
+USER root
+
+RUN apt -y update
+RUN apt install -y make wget
+
+# Initialize nvm
+RUN wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash
+RUN source /root/.bashrc && nvm install --lts
+
+# Initialize go
+RUN wget -O go.tar.gz https://go.dev/dl/go1.23.1.linux-amd64.tar.gz
+RUN rm -rf /usr/local/go && tar -C $HOME -xzf go.tar.gz
+RUN rm -rf go.tar.gz
+RUN echo "export PATH=$PATH:$HOME/go/bin" >> $HOME/.bashrc
+
+# Install python
+RUN wget -O python.tar.xz https://www.python.org/ftp/python/3.12.6/Python-3.12.6.tar.xz
+RUN tar -xf python.tar.xz -C $HOME 
+RUN chmod +x $HOME/Python-3.12.6/configure
+RUN /$HOME/Python-3.12.6/configure
```

Comment:
That's one way to go about it. The other is to pull `pyenv` and install Python in a similar way how `nvm` installs the `lts` node. I think the latter will give you more flexibility but I don't know if that's needed in the first place. 

I will have @IvanObreshkov have the last saying here. My suggestion is that you just add a comment saying 

```
# You could swap this for `pyenv` to enable easily hopping between python versions
``` 

or something of the sorts.